python/lsst/sattle/app/main.py

```diff
@@ -18,16 +19,23 @@ def read_tle_from_url(url, tle_source):
 
     # TODO: Currently this requires manual input of ra/dec. Make this read
     # from a file so it can be checked faster.
-    if tle_source == 'satchecker':
+
+    # We need to change this to be more useful for verification.
+    # Should make it a per day tle list, so verification checks per day.
+    # This would mean we remove the initial query though this is important
+    # Keep this in but change
+    # Needs to be all satellites visible in a night
```

Comment:
Please tidy up the comments and make a TODO with a ticket number if necessary

Reply:
Sorry some of this, specifically the improvements to the sattle work, are actually being improved on a separate ticket. I'll remove comments and clean up for this specific ticket, but the work will be done elsewhere.